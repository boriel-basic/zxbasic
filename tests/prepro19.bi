' Must return
' PRINT (10 + PRINT 20;);

#define macro(x) PRINT x;

macro((10 + macro(20)))


