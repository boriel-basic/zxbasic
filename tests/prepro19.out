




PRINT (10 + PRINT 20;);


